src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+import NRVulnManagementMain from 'images/vulnerability-management_screenshot-crop_main.webp'
+
+
+Welcome to the New Relic observability platform! Our platform gives you deep visibility into your systems. It helps you solve problems you may not have been aware of, regardless of the technology you use. You can see how New Relic works [here](#how-it-works) or [get started](#get-started) now if you want to jump right in.
+
+New Relic provides many ways to improve visibility. They let you monitor, troubleshoot, and improve digital performance. Here's a brief overview of our most popular capabilities and features.
+
+
+# Alerts [#alerts]
+<SideBySide>
+  <Side>
+Alerts help you identify issues that look unusual in your applications and code. New Relic includes some <InlinePopover type="alerts" /> when you install it, and you can also create your alerts. You receive alerts through tools like PagerDuty, ServiceNow, Jira, Slack, and more. Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue. 
+  
+Learn more about alerts [here](/docs/alerts-applied-intelligence/overview/).
+
+  </Side>
+  <Side>
+    <img
+      title="New Relic alerts"
+      alt="New Relic alerts"
+      src={NRAlertsMain}
+    />
+  </Side>
+  </SideBySide>
+
+# APM [#apm]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic APM"
+    alt="New Relic APM"
+    src={NRApmMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic instruments your application by using one of our many language agents. These agents collect metrics from your application and then send the metrics to our <InlinePopover type="apm" />. It lets you monitor your application using pre-built dashboards. 
+  
+  Learn more about APM [here](/docs/apm/new-relic-apm/getting-started/introduction-apm/).
+
+  </Side>
+  </SideBySide>
+
+# Browser [#browser]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="browser" /> capability collects data from the navigation timings in all end users' browsers. It sends that data directly to New Relic while blending with the service that served the requests. When you load that page URL, you can easily see which back-end transaction fulfilled that request. 
+
+  Learn more about browser [here](/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic browser"
+  alt="New Relic browser"
+  src={NRBrowserMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Dashboards [#dashboards]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic dashboards"
+  alt="New Relic dashboards"
+  src={NRDashboardsMain}
+/>
+
+  </Side>
+  <Side>
+
+  Our UI allows you to create custom charts and share them easily. Our <InlinePopover type="dashboards" /> capability lets you collect and visualize your data preferences across our platform. Tailor your charts to display the information you need. Access and analyze your data from any location within our platform.
```

Comment:
```suggestion
  New Relic lets you organize your data how you want with our <InlinePopover type="dashboards" />. Tailor your charts to display the data that matters to you, then access those charts from any location within our platform.
```

Reply:
Dashboards are a little tricky tbh. I'm not even convinced by this edit... I feel like there's something missing... Dashboards are used to tell specific stories by whatever ~account or ~team builds them out, and we also offer a kind of dashboard with install.

I've heard SMEs say that sometimes customers'll build dashboards around specific kinds of errors, or teams that manage them, or by event or tech type... I think you convey that with "data that matters to you" but it seems almost too vague. 

My only other recommendation is to sidestep referring to the UI explicitly and keep the actor "New Relic." Our UI is just a design element, but it doesn't "do" anything. An easy way to avoid referrring to the UI is to just say new relic lets you do X thing

Reply:
Great explanation, thanks! I've rewritten all:
With New Relic's dashboards, you can arrange your data how you want. Tailor <InlinePopover type="dashboards" /> for your application and system performance info. Adjust charts to show your key data and access these charts easily across our platform. This adaptability helps you monitor and study your data efficiently.